src/components/Label/Label.module.scss

```diff
@@ -10,6 +10,10 @@
   border-radius: var(--border-radius-small);
   padding: 1px var(--spacing-small);
   position: relative;
+
+  &.clickable {
+    cursor: pointer;
```

Comment:
Also our `focus-style` is needed here - all together can be achieved by importing Clickable mixin as done in `Chips`
```
@use "../Clickable/Clickable.module" as clickable;
@include clickable.clickable;
```

Reply:
Oh that's important 🙏🏼